index.js

```diff
@@ -1,7 +1,53 @@
 // Iteration 1: Names and Input
 
+let hacker1 = 'Francisco';
+
+console.log(`The driver's name is ${hacker1}`);
+
+let hacker2 = 'Rafaela';
+
+console.log(`The navigator's name is ${hacker2}`);
+
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length){
```

Comment:
Here, you could've created a variable called` driver = hacker1.length `and another called` navigator = hacker2.length `to make it more readable!